contrib/lorri.nix

```
@@ -0,0 +1,38 @@
+{ config, systemd, ... }:
+
+let
+  lorri = import
+    (fetchTarball "https://github.com/target/lorri/tarball/rolling-release")
+    { };
+  socketUnit = "lorri";
+  socketPath = "lorri/daemon.socket";
+in {
+  config = {
+    systemd.user.sockets.${socketUnit} = {
+      description = "Socket for Lorri Daemon";
+      enable = true;
+      wantedBy = [ "sockets.target" ];
+      socketConfig = {
+        ListenStream = "%t/${socketPath}";
+        RuntimeDirectory = "lorri";
+      };
+    };
+
+    systemd.user.services.lorri = {
+      description = "Lorri Daemon";
+      requires = [ "${socketUnit}.socket" ];
+      after = [ "${socketUnit}.socket" ];
```

```suggestion
      after = [ "${socketUnit}.socket" ];
      path = with pkgs; [ nix ];
```
 
probably need some others too.